docs/konnectors_oauth.md

```diff
@@ -0,0 +1,149 @@
+# Doctypes
+
+`io.cozy.konnectors` gives their desiderata for an account
+
+```json
+{
+  "fields": {
+    "account": {
+      "doctype": "io.cozy.accounts",
+      "account_type": "trainlines",
+      "accountFormat": "login,password",
+    }
+  },
+
+}
+```
+
+`io.cozy.accounts` contains authentification information for an account
+
+```json
+{
+    "name" : "Gmail Perso",
+    "account_type": "google",
+    "status": "connected",
+    "auth": {
+      "user": "my-personal-account@gmail.com",
+      "password": "my-secret"
+    }
+}
+```
+
+`io.cozy.accounts_type` contains the oauthconfiguration
+
+```json
+{
+  "_id":"google",
+  "mode": "userpass"
+}
+```
+
+```json
+{
+  "_id":"service.example",
+  "mode": "oauth-server-flow",
+  "client_id": "",
+  "client_secret": "",
+  "auth_endpoint": "https://service.example/auth",
+  "token_endpoint":  "https://api.service.example/token"
+}
+```
+
+@TODO some encryption of cozy client secret might be useful
+
+# Reminder OAuth flows
+
+**Service** is the website konnector aims to retrieve informations from
```

Comment:
s/konnector aims to retrieve informations from/from which konnector aims to retrieve informations/